test/indexer/test_hca_indexer.py

```diff
@@ -102,6 +102,9 @@ class BundleAndSize(NamedTuple):
                                      size=258)
         self.assertTrue(small_bundle.size < IndexWriter.bulk_threshold < large_bundle.size)
 
+        from azul.plugin import Plugin
```

Comment:
If this is to avoid a circular import, I would like to talk about avoiding it. Otherwise, it should be top-level import.

Reply:
… talk about eliminating it

Reply:
Should be top-level import. Will fix.